interface/themes/core/patient/demographics.scss

```diff
@@ -34,3 +35,116 @@
     }
   }
 }
+
+
+#DEM table {
+  width: 100%;
+}
+
+#DEM .label_custom {
+  text-align: right;
```

Comment:
This is what makes form look weird. Try auto however due to the way LBF renders using table columns I doubt you'll be able to get this to look the way you hope i.e data not proportional in view. I think until we get rid of the tables in this view we'll go in circles:)